fe/fe-core/src/main/java/com/starrocks/catalog/ExternalOlapTable.java

```diff
@@ -153,24 +152,27 @@ public void parseFromProperties(Map<String, String> properties) throws DdlExcept
     }
 
     private long dbId;
+    private long sourceDbId;
```

Comment:
dbId and sourceDbId are confused, better to use sourceDbId as db id of remote table?
Why not put the remote db id in the ExternalTableInfo?